private/bufpkg/bufmodule/bufmodulebuild/module_file_set_builder.go

```diff
@@ -59,12 +57,8 @@ func (m *moduleFileSetBuilder) build(
 	workspace bufmodule.Workspace,
 ) (bufmodule.ModuleFileSet, error) {
 	var dependencyModules []bufmodule.Module
-	hashes := make(map[string]struct{})
-	moduleHash, err := protoPathsHash(ctx, module)
-	if err != nil {
-		return nil, err
-	}
-	hashes[moduleHash] = struct{}{}
+	moduleIds := make(map[string]struct{})
+	moduleIds[module.ID()] = struct{}{}
 	if workspace != nil {
 		// From the perspective of the ModuleFileSet, we include all of the files
```

Comment:
This long comment I wrote is likely now out of date - the same level of explanation needs to be here for what we're currently choosing, and alternatives.